Maturin compilation issue

`maturin build --cargo-extra-args="--features python-module"` fails with error:

```
error[E0308]: mismatched types=========> ] 83/84: fbleau   
   --> src/python_module.rs:103:13                                                                                                                                                                                
    |                                              
103 |             train_x,                                                                                                                                                                                        
    |             ^^^^^^^ expected struct `ArrayBase`, found struct `ndarray::ArrayBase`
    |                             
    = note: expected struct `ArrayBase<OwnedRepr<f64>, Dim<[usize; 2]>>`
               found struct `ndarray::ArrayBase<ndarray::data_repr::OwnedRepr<f64>, ndarray::dimension::dim::Dim<[usize; 2]>>`
    = note: perhaps two different versions of crate `ndarray` are being used?
                                                                              
                                                        
error[E0308]: mismatched types=========> ] 83/84: fbleau
   --> src/python_module.rs:104:13
    |                             
104 |             train_y, 
    |             ^^^^^^^ expected struct `ArrayBase`, found struct `ndarray::ArrayBase`
    |                               
    = note: expected struct `ArrayBase<OwnedRepr<usize>, Dim<[usize; 1]>>`
               found struct `ndarray::ArrayBase<ndarray::data_repr::OwnedRepr<usize>, ndarray::dimension::dim::Dim<[usize; 1]>>`
    = note: perhaps two different versions of crate `ndarray` are being used?
                                                 
                                                  
error[E0308]: mismatched types=========> ] 83/84: fbleau                                          
   --> src/python_module.rs:105:13       
    |
105 |             test_x,
    |             ^^^^^^ expected struct `ArrayBase`, found struct `ndarray::ArrayBase`
    |                             
    = note: expected struct `ArrayBase<OwnedRepr<f64>, Dim<[usize; 2]>>`
               found struct `ndarray::ArrayBase<ndarray::data_repr::OwnedRepr<f64>, ndarray::dimension::dim::Dim<[usize; 2]>>`
    = note: perhaps two different versions of crate `ndarray` are being used?

                                                        
error[E0308]: mismatched types=========> ] 83/84: fbleau
   --> src/python_module.rs:106:13
    |                                                         
106 |             test_y,                                                                                                                                                                                         
    |             ^^^^^^ expected struct `ArrayBase`, found struct `ndarray::ArrayBase`
    |
    = note: expected struct `ArrayBase<OwnedRepr<usize>, Dim<[usize; 1]>>`
               found struct `ndarray::ArrayBase<ndarray::data_repr::OwnedRepr<usize>, ndarray::dimension::dim::Dim<[usize; 1]>>`
    = note: perhaps two different versions of crate `ndarray` are being used?


error: aborting due to 4 previous errors ] 83/84: fbleau


For more information about this error, try `rustc --explain E0308`.

error: could not compile `fbleau`
```

Probably due to this issue: https://githubmemory.com/repo/PyO3/rust-numpy/issues/188.

Quick fix: use dependency `ndarray = "0.15"` in `Cargo.toml`.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,7 +31,7 @@ csv = "1.1"
 ordered-float = "2.0"
 float-cmp = "0.8"
 strsim = "0.10.0"
-ndarray = "0.14.0"
+ndarray = "0.15"
 pyo3 = { version = "0.13.2", features = ["extension-module"], optional = true }
 numpy = { version = "0.13", optional = true }
 

